agent_build_refactored/managed_packages/scalyr_agent_python3/internal/agent-python3-config.py

```diff
@@ -0,0 +1,223 @@
+# Copyright 2014-2023 Scalyr Inc.
```

Comment:
This is not a new file, it's basically just a copy of the previous version of the  `agent_build_refactored/managed_packages/scalyr_agent_python3/agent-python3-config` file. The problem is that this python script can not be executed directly anymore, and it has to be executed from the "wrapper" shell script.
But since it is a "visible" executable and can be used by a user, the wrapper has to have the same name, creating this huge diff.